research/schooltests/sql.txt

```diff
@@ -0,0 +1,126 @@
+
+-------------------------------
+
+Erstelle eine Übersicht aller Playlisten mit mehr als 10 Songs, gruppiert nach dem/der Ersteller/in. Gib dabei den Namen und den/die Ersteller/in aus, sowie die Anzahl aus.
```

Comment:
- first "aus"

"Erstelle eine Übersicht aller Playlisten mit mehr als 10 Songs, gruppiert nach dem/der Ersteller/in. Gib dabei den Namen und den/die Ersteller/in, sowie die Anzahl aus."

I think "gruppiert nach" is here okay, cause core is the having clause

Reply:
done